specification/search/data-plane/Microsoft.Azure.Search.Service/preview/2017-11-11-preview/searchservice.json

```diff
@@ -4183,7 +4187,7 @@
 		"externalDocs": {
 			"url": "https://docs.microsoft.com/azure/search/cognitive-search-tutorial-blob"
 	    },
-		"description": "A list of Cognitive Search skills."
+		"description": "A list of cognitive skills."
```

Comment:
is this the right description?
only indexers have datasource, skillset and index names

Reply:
Synced up with Victor offline on this.